### ISSUE TITLE
[Bug]: LC0095 false positive with ErrorInfo objects

### ISSUE THREAD

**Issue by TKapitan** [ALCops/Analyzers#263]
### Analyzer

LinterCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

LC0095

### Description

Procedures added as actions to ErrorInfo objects (ErrorInfo.AddAction) must have ErrorInfo parameter. It's similar to interface - they should have the parameter even if not used.

### AL code to reproduce

```al
    procedure OpenBusinessHubSetup(ErrorInfo: ErrorInfo)
    var
        BusinessHubSetup: Page FS5A29BusinessHubSetup;
    begin
        BusinessHubSetup.Run();
    end;

    procedure DisableBusinessHub()
    var
        ...
    begin
        ...
        ErrorInfo.AddAction(ViewBusinessGroupsLbl, Codeunit::FS5A29BusinessHubMgt, 'OpenBusinessGroupsList');
        Error(ErrorInfo);
    end;
```

### Additional context

<img width="1231" height="296" alt="Image" src="https://github.com/user-attachments/assets/c51dfddd-28a6-4b50-bbf5-5f3f50bf1be9" />

if the parameter is removed, the AddAction shows AL0821

<img width="1266" height="192" alt="Image" src="https://github.com/user-attachments/assets/d660518c-173e-4b1b-aedc-1898270a369d" />

**Comment by Arthurvdv** [comment]
Thanks, I've resolved this in the release of `v0.8.2` (and next to ErrorInfo I've also excluded the Notification param).

I'll close the issue here, but feel free to reopen if it's not working as expected.
